### DISCUSSION TITLE
Ohne HACSS ?

### DISCUSSION THREAD

**Discussion by Sv4en** [cathiele/homeassistant-goecharger#58] [Q&A]
Hallo
Gibt es eine Möglichkeit ohne HACSS den goecharger zu benutzen?

Mit HACSS läuft bei mir die vicare Integration nicht
Als AddOn z.B.

Schönen Abend

**Comment by cathiele** [comment]
Du kannst einfach das Addon in dein custom_components Verzeichnis kopieren.
